StitchCore-iOS/StitchCore-iOS/Core/StitchAppClient.swift

```diff
@@ -39,7 +43,10 @@ public protocol StitchAppClient {
      *            passed in the `forProvider` parameter.
      */
     func serviceClient<T>(forService provider: AnyServiceClientProvider<T>) -> T
-    
+
+    // MARK: Functions
+
+    // swiftlint:disable line_length
```

Comment:
[minor] We should avoid disabling this check at all costs. The method sign below could have newline between parameters.

Reply:
it's like that because jazzy cannot get the full function signature with newlines in protocols :(

Reply:
Ah wow, that seems like a pretty huge bug.